src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsViewController.swift

```diff
@@ -91,10 +91,13 @@ final class ExposureSubmissionSymptomsViewController: DynamicTableViewController
 			switch index {
 			case 0:
 				selectedSymptomsOption = .yes
+				navigationFooterItem?.primaryButtonTitle = AppStrings.ExposureSubmissionSymptoms.continueButton
 			case 1:
 				selectedSymptomsOption = .no
+				navigationFooterItem?.primaryButtonTitle = AppStrings.ExposureSubmissionSymptomsOnset.continueButton
```

Comment:
Would be nice to have this as a separate AppString for this Screen, e.g. `AppStrings.ExposureSubmissionSymptoms.doneButton`

Reply:
should i change the name of continue button or add a new variable for this? I'm not sure why it was called continue

Reply:
I would propose to add a new one called `AppStrings.ExposureSubmissionSymptoms.doneButton` that contains the "Fertig"/"That's it!" string, and keep the old one `AppStrings.ExposureSubmissionSymptoms.continueButton` that contains the "Weiter"/"Next" string.

Reply:
I changed it actually, for me doesn't make sense to have a "continue" button that says "done", or what do you think?